deploy/examples/cluster.yaml

```diff
@@ -281,6 +281,31 @@ spec:
     # No values or 0 means that the operator will wait until the placement groups are healthy before unblocking the next drain.
     pgHealthCheckTimeout: 0
 
+  # csiDriverOptions defines defines CSI Driver settings applied per cluster.
+  csiDriverOptions:
+    # common settings for all CSI drivers.
+    common:
+      readAffinity:
+        # set enabled to true to enable read affinity.
+        enabled: false
+        # CRUSH location labels define which node labels to use
+        # as CRUSH location. This should correspond to the values set in
+        # the CRUSH map.
+        # Defaults to all the labels mentioned in
+        # https://rook.io/docs/rook/latest/CRDs/Cluster/ceph-cluster-crd/#osd-topology
+        # crushLocationLabels:
```

Comment:
I don't know why anyone would ever need to override this since if they care about topology, they should already be using the same topology labels that rook already uses. So I'd suggest we not even include the crushLocationLabels in the cluster.yaml example, we can just have it in the schema and the docs.

Reply:
I wonder if we should also just strike this from the schema entirely and make it so users cannot modify this. It seems to me that it might be best if Rook handles this via its own mechanisms internally. With this in the schema, it seems more likely that a user might set this in such a way that drastically conflicts with what Rook has done with the Ceph cluster. But perhaps there is a use for this that I don't quite understand.